### ISSUE TITLE
AWQ Support 

### ISSUE THREAD

**Issue by allzero-kwon** [vllm-project/vllm#1488]
I got this error to run awq model with VLLM on a v100 gpu.
How can i resolve this ? 

```
The quantization method awq is not supported for the current GPU. Minimum capability: 75. Current capability: 70.
```

**Comment by casper-hansen** [comment]
It is recommended to use a GPU with the Ampere architecture. V100 is not supported.

**Comment by WoosukKwon** [comment]
Closed as it is a duplicate of #1345
